packages/editor/src/components/post-template/block-theme.js

```diff
@@ -53,7 +53,11 @@ export default function BlockThemeControl( { id } ) {
 
 	const canCreateTemplate = useSelect(
 		( select ) =>
-			select( coreStore ).canUser( 'create', 'templates' ) ?? false
+			select( coreStore ).canUser( 'create', {
+				kind: 'postType',
+				name: 'wp_template',
+			} ),
```

Comment:
Should we keep the default: `?? false`? Theoretically `canUser` can still return `undefined`, right?

Reply:
Resolved in [e683bab](https://github.com/WordPress/gutenberg/pull/63415/commits/e683babcd050b2efa244fa8493b6759da0389a7f).